admin_manual/installation/apps_management_installation.rst

```diff
@@ -51,37 +35,31 @@ your ownCloud Admin page, Personal page, or in ``config.php``.
 Adding Third Party Apps
 -----------------------
 
-Some apps are developed and supported by ownCloud directly, while other apps are 
-created by third parties and available for your ownCloud server installation.  
-Apps developed by the ownCloud community show a *recommended* designation. Any 
-apps that are not developed by ownCloud but have been reviewed by the ownCloud 
-security team show a *3rd party* designation. Install unsupported apps at your 
-own risk.
+Some apps are developed and supported by ownCloud directly. These have an 
+**Official** tag. Apps with the **Approved** tag are community-developed and 
+supported; they are maintained by trusted developers, and are under active 
+development. Only **Official** and **Approved** apps are linked on the Apps 
+page.
 
 To understand what an application does, you can click the app name to view a 
 description of the app and any of the app settings in the Application View 
-field.  Clicking the *Enable* button will enable the app.  If the app is not 
+field.  Clicking the **Enable** button will enable the app.  If the app is not 
 part of the ownCloud installation, it will be downloaded from the app store, 
 installed and enabled. 
 
-You can view new, unreviewed or unstable applications in the 
-`ownCloud Apps Store <https://apps.owncloud.com/>`_.
+You can view new, unreviewed or unstable applications in the `ownCloud Apps 
+Store <https://apps.owncloud.com/>`_. Install unsupported apps at your own risk.
```

Comment:
IIRC its there since 8.2 with the introduction of the "experimental" apps. With 8.2+ a user don't need to browse the apps.owncloud.com anymore. AFAIK that was also the reason why the "More apps..." link was removed.